openpype/tools/loader/widgets.py

```diff
@@ -1033,6 +1033,18 @@ class FamilyListView(QtWidgets.QListView):
     def __init__(self, dbcon, family_config_cache, parent=None):
         super(FamilyListView, self).__init__(parent=parent)
 
+        from Qt import QtCore, QtWidgets
+        try:
+            settings = QtCore.QSettings("OpenPype", "FamilyList")
+        except Exception:
+            settings = None
```

Comment:
I'm pretty sure settings like these should be stored differently. @iLLiCiTiT likely has some decent input on how to expose this (and potentially even beyond the scope of just this setting).

Also, I'm pretty sure that using the "settings" value might interfere with e.g. the feature in OpenPype where you can define which default families are visible per host, etc. There's something in settings for that. (We prefer to actually have different families visible per host instead of a global setting value for a machine)

Reply:
> Also, I'm pretty sure that using the "settings" value might interfere with e.g. the feature in OpenPype where you can define which default families are visible per host, etc. There's something in settings for that. (We prefer to actually have different families visible per host instead of a global setting value for a machine)

Can you tell me where is the option to define which family is visible in the loader? 
I don't think there is any interference with the other options because @FadyFS tries to store the checked families in the settings to put the checked back on when opening, it doesn't filter them.

Reply:
> I don't think there is any interference with the other options because @FadyFS tries to store the checked families in the settings to put the checked back on when opening, it doesn't filter them.

That's exactly what the filters do too... `project_settings/global/tools/loader/family_filter_profiles`

Reply:
> It's slightly different because the implementation of fady allows to remember the family check during the last sessions.

That's exactly what I tried to mention - is that if the last session overrides the settings, then basically the settings only ever apply once and don't differentiate between default visible families presets of different hosts.